monitor/monitor_test.go

```diff
@@ -0,0 +1,197 @@
+package monitor
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/gofiber/fiber/v3"
+
+	"github.com/valyala/fasthttp"
+)
+
```

Comment:
Replace all the calls to `AssertEqual()` with `require.Equal()` from https://github.com/stretchr/testify

Reply:
[b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8](https://github.com/Redish101/contrib/commit/b5c4678b7d76ec59aae75fd0e492f0388e6ab5d8)